examples/smac/feedforward/decentralised/run_ippo_smac_death_masking.py

```diff
@@ -73,15 +74,16 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
 
     # Optimisers.
     policy_optimiser = optax.chain(
-        optax.clip_by_global_norm(40.0), optax.scale_by_adam(), optax.scale(-1e-4)
+        optax.clip_by_global_norm(40.0), optax.scale_by_adam(), optax.scale(-5e-4)
     )
 
     critic_optimiser = optax.chain(
-        optax.clip_by_global_norm(40.0), optax.scale_by_adam(), optax.scale(-1e-4)
+        optax.clip_by_global_norm(40.0), optax.scale_by_adam(), optax.scale(-5e-4)
     )
 
     # Create the system.
     system = ippo.IPPOSystem()
+    system.update(HuberValueLoss)
```

Comment:
Do we need Huber loss when testing the death masking code? Dit is not work with vanilla IPPO?